README.md

```diff
@@ -13,7 +13,7 @@ Roblox-js is a node module that provides an interface for [ROBLOX](http://www.ro
 
 Most functions are related to group service but there are other general functions as well. The list of main functions is in the contents section, they all have detailed documentation.
 
-To use this with HttpService simply set up API's on your node server for accessing the functions, you can use the example server outlined in the below example server section as a base. Note that this does not use the latest version of the module and it is recommended you learn how to use the library directly.
+To use this with HttpService simply set up API's on your node server for accessing the functions, you can use the example server outlined in the example server section as a base. Note that this does not use the latest version of the module and its recommended you learn how to use the library directly.
```

Comment:
Changing "it is" to "its" introduces a grammatical error. The proper form would be "it's" and it is not so critical as to necessitate a pull request.

Reply:
Meant to put "it's".